### PR TITLE
Implement design from wireframes

### PR DESCRIPTION
**Pull request by kdeloach**
Changes:

* Implement tree UI from wireframes
* Add info box
* Add layer menu

image

### CODE REVIEW

File: src/GeositeFramework/plugins/layer_selector_v2/fontello-8ae227d1/demo.html [hunk 1]
```diff
@@ -0,0 +1,322 @@
+<!DOCTYPE html>
```
Comment (caseypt) [inline src/GeositeFramework/plugins/layer_selector_v2/fontello-8ae227d1/demo.html]: Do we need this file?

File: src/GeositeFramework/plugins/layer_selector_v2/main.js [hunk 2]
```diff
@@ -51,25 +51,96 @@ define([
...
+                        var $el = $(this),
+                            $parent = $el.closest('li'),
+                            layerId = $parent.attr('data-layer-id'),
+                            $menu = self.createLayerMenu(layerId),
+                            $shadow = self.createLayerMenuShadow(),
+                            pos = $el.offset();
+                        $menu.css({
+                            top: pos.top,
+                            left: pos.left
+                        });
```
Comment (caseypt) [inline src/GeositeFramework/plugins/layer_selector_v2/main.js]: Consider moving all of the menu setup into a function.

File: src/GeositeFramework/plugins/layer_selector_v2/main.js [hunk 3]
```diff
@@ -160,16 +231,27 @@ define([
             renderTree: function() {
                 var html = this.treeTmpl({
                     layers: this.state.getLayers(),
-                    renderLayer: _.bind(this.renderLayer, this)
+                    renderLayer: _.bind(this.renderLayer, this, 0)
```
Comment (caseypt) [inline src/GeositeFramework/plugins/layer_selector_v2/main.js]: Did not know that `_.bind` could also do partials.

File: src/GeositeFramework/plugins/layer_selector_v2/main.js (hunk 2 above)
Comment (caseypt) [inline src/GeositeFramework/plugins/layer_selector_v2/main.js]: Additionally, this is may be out of the scope of this issue, but I think we should switch to just calling functions in the event handlers and doing all the work there. `bindEvents` is getting long.

**Comment by caseypt** [comment]
+1. I've been working on this branch all day, and the style looks really good. The CSS looks good to me, but I'll defer to @jfrankl.

File: src/GeositeFramework/plugins/layer_selector_v2/main.js (hunk 2 above)
Comment (caseypt) [inline src/GeositeFramework/plugins/layer_selector_v2/main.js]: We could also probably abstract out a function for getting the nearest `layerId`.

Comment (caseypt) [inline src/GeositeFramework/plugins/layer_selector_v2/main.js]: I'll do this in my rebase of #539

**Comment by kdeloach** [comment]
Added a commit to include feedback from Jeff.

File: src/GeositeFramework/plugins/layer_selector_v2/main.js (hunk 2 above)
Comment (kdeloach) [inline src/GeositeFramework/plugins/layer_selector_v2/main.js]: Agreed on all points.

File: src/GeositeFramework/plugins/layer_selector_v2/fontello-8ae227d1/demo.html (hunk 1 above)
Comment (kdeloach) [inline src/GeositeFramework/plugins/layer_selector_v2/fontello-8ae227d1/demo.html]: I didn't delete any of the files from the fontello zip.

**Pull request merged**